util/command_parser.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"flag"
+	"fmt"
+)
+
+// ParseCommand parses and check user input.
+// It returns a map if input is valid.
+func ParseCommand(command []string) (map[string]string, error) {
```

Comment:
Thank you. I've fixed.

ParseCommand might not be good function name because Command has some meanings in sd-cmd context and it might not easy to find the appropriate meaning. I concerned about that and I adopted `parseInputCommand` .